### PR TITLE
Autotester: Now remembers the previous folder.

### PR DESCRIPTION
**Pull request by NissimHadar**
In Autotester, the program now cache that recent folder opened instead of restarting from a default root folder.
This greatly speeds up the use of the auto-tester when creating multiple tests.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9641/app_13139_PR_9d52306ac599fdcecd319ee61fc2c043e0a1771d.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9641/HighFidelity-Beta-PR13139-9d52306ac599fdcecd319ee61fc2c043e0a1771d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9641/HighFidelity-Beta-Interface-PR13139-9d52306ac599fdcecd319ee61fc2c043e0a1771d.exe

File: tools/auto-tester/src/Test.h [hunk 1]
```diff
@@ -89,6 +89,12 @@ class Test {
     const int NUM_DIGITS { 5 };
     const QString EXPECTED_IMAGE_PREFIX { "ExpectedImage_" };
 
+    // We have to directories to work with.
```
Comment (samcake) [inline tools/auto-tester/src/Test.h]: typo: two instead of 'to'

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9754/HighFidelity-Beta-PR13139-a9a783588bae2dca4f048fd257253f4cca9dfe71.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9754/HighFidelity-Beta-PR13139-a9a783588bae2dca4f048fd257253f4cca9dfe71.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9754/HighFidelity-Beta-Interface-PR13139-a9a783588bae2dca4f048fd257253f4cca9dfe71.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9754/HighFidelity-Beta-PR13139-a9a783588bae2dca4f048fd257253f4cca9dfe71.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9754/HighFidelity-Beta-Interface-PR13139-a9a783588bae2dca4f048fd257253f4cca9dfe71.exe

**Comment by samcake** [comment]
This PR is changing the autotester tool only which is used internally, not exposed to public users
We (Engine team) have qaed successfully and we are taking responsibility for merging this PR.
Thanks

**Pull request merged**